app/components/agenda/agendaitem/agendaitem-postponed.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import addWeeks from 'date-fns/addWeeks';
+import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+
+/**
+ * @argument subcase
+ * @argument meeting
+ * @argument agendaActivity
+ */
+export default class AgendaitemPostponed extends Component {
+  @service store;
+  @service agendaService;
+
+  @tracked canPropose = false;
+  @tracked modelsForProposedAgenda;
+  @tracked newMeeting;
+
+  constructor() {
+    super(...arguments);
+    this.loadProposedStatus.perform();
+    this.loadProposableMeetings.perform();
```

Comment:
This should only be executed if the agendaitem is not reproposed yet? Can it be embedded in the `loadProposedStatus` task? 

Something like
```javascript
if (latestActivity) { // agendaitem is already proposed again
   // load model to show link to new agendaitem
} else {
  // load proposable meetings
} 
```